test/e2e/step-definitions/healthdeclaration-steps.ts

```diff
@@ -25,6 +26,10 @@ Before({ tags: '@catc1e' }, () => {
   this.testCategory = 'c';
 });
 
+Before({ tags: '@catce' }, () => {
+  this.testCategory = 'c';
```

Comment:
shouldn't category be ce based on the tag

Reply:
This is because on this page the HTML uses the test category  'c' on the health declaration page, but due to how hooks has been used to drive other parts of the test its c1e on other pages. This is going to be removed when the Page Object Model is implemented.